.devcontainer/on-create.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# install k3d
+wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v5.4.4 bash
```

Comment:
Any chance that we can rebuild the codespaces `asb` image to include the k3d codespace image? This way, we wouldn't have to install k3d manually

Reply:
We do have the `FROM ghcr.io/cse-labs/k3d:latest` image from cse-labs.

Reply:
I think the ingress certificates are loaded into the image so we would need to be careful on not overwriting them.

Reply:
I'll be adding this to triage. The image we're using for this repo's codespace is old. It was probably setup before we started to standardize on the cse-labs images. I took the path of least resistance here to avoid breaking any existing dependencies.

Reply:
added a task for us to triage https://github.com/retaildevcrews/wcnp/issues/910.